examples/deployment/scripts/transfer/gold_sc.js

```diff
@@ -0,0 +1,27 @@
+const {
+  transferMicroAlgosContract
+} = require('../src/asa-helpers');
+
+async function run(runtimeEnv, accounts, deployer) {
+
+  const goldOwnerAccount = deployer.accountsByName.get("gold-owner-account");
+  const johnAccount = deployer.accountsByName.get("john-account");
+
+  if (deployer.isDeployMode) {
+
+    const logicSignature = deployer.asc.get("3-gold-asc.teal").logicSignature;
+
+    // ALGO transfer
+
+    // Will pass - As according to .teal logic, amount should be <= 100
+    // Transaction PASS
+    await transferMicroAlgosContract(deployer, goldOwnerAccount, johnAccount, 50, logicSignature);
+
+    // Gets rejected by logic - As according to .teal logic, amount should be <= 100
+    // Transaction FAIL
+    await transferMicroAlgosContract(deployer, goldOwnerAccount, johnAccount, 150, logicSignature);
```

Comment:
please use `try .. catch` with `console.error` in catch section

Reply:
ok